src/audio/kpb.c

```diff
@@ -43,6 +43,7 @@
 
 /* KPB private data, runtime data */
 struct comp_data {
+	uint64_t state_log;
```

Comment:
Do we need 64 bits?

Reply:
Wider data size gives us the possibility to log more states. So I would keep it as 64 bits.